package.json

```diff
@@ -197,6 +202,7 @@
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-simple-import-sort": "^7.0.0",
     "fast-json-stable-stringify": "^2.1.0",
+    "jest": "^27.5.1",
```

Comment:
This should already be included as part of react-scripts.

Reply:
removed.